.snippets/text/_common/async-backing-moonbase.md

```diff
@@ -1,2 +1,2 @@
 !!! note
-    As of runtime 2800, [asynchronous backing](https://wiki.polkadot.network/docs/learn-async-backing){target=\_blank} has been enabled on Moonbase Alpha. As a result, the target block time was reduced from 12 seconds to {{ networks.moonbase.block_time }} seconds, which may break some timing-based assumptions. Note that this is for Moonbase Alpha only.
+    As of runtime 2800, [asynchronous backing](https://wiki.polkadot.network/docs/learn-async-backing){target=\_blank} has been enabled on Moonbase Alpha. As a result, the target block time was reduced from 12 seconds to 6 seconds, which may break some timing-based assumptions. Note that this is for Moonbase Alpha only.
```

Comment:
Why not use a variable?

Reply:
they don't render correctly in the snippets